index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Majd Portfolio</title>
+    <link rel="stylesheet" href="./styles/main.css" />
+  </head>
+  <body>
+    <header>
+      <div class="container">
+        <div class="icon">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="container">
+        <h2>
+          <span>Hey there. I’m Majd. </span>
+          <span> I’m a software developer</span>
+        </h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+        <div class="icons">
+          <div>
+            <a
+              href="https://www.linkedin.com/in/majdalkilany/"
+              target="”_blank”"
+            >
+              <svg
+                width="20"
+                height="21"
+                viewBox="0 0 20 21"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M13.6444 3.86914C10.1556 3.86914 7.28857 6.64258 7.28857 10.0925V19.8333C7.28857 20.2054 7.59946 20.5098 7.97942 20.5098H11.5718C11.9518 20.5098 12.2627 20.2054 12.2627 19.8333V10.363C12.2627 9.61895 12.8844 9.01014 13.6444 9.01014C14.4043 9.01014 15.0261 9.61895 15.0261 10.363V19.8333C15.0261 20.2054 15.3369 20.5098 15.7169 20.5098H19.3093C19.6893 20.5098 20.0001 20.2054 20.0001 19.8333V10.0925C20.0001 6.6764 17.1677 3.86914 13.6444 3.86914Z"
```

Comment:
Kindly update your list of social links to be wrapped in a `<ul>` tag instead of a `<div>` tag. This is for semantic reasons as ul tags are used to wrap most lists in a webpage

- In addition to that, please wrap each img in a `<li>` tag and make sure that the images are all links using an `<a>` tag. Your final code should look similar to something like this:

`<ul>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
`</ul>`